lib/Wikia/src/Util/SamplerProxyBuilder.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Wikia\Util;
+
+class SamplerProxyBuilder {
+
+	protected $enableShadowing;
+	protected $methodSamplingRate;
+
+	protected $originalCallable;
+	protected $alternateCallable;
+	protected $resultsCallable;
+
+	public function enableShadowing( $isEnabled ) {
```

Comment:
Since this is a setter, I suggest to either rename this method to 
setEnableShadowing( $isEnabled )
or split it into parameter-less
enableShadowing
disableShadowing

I suggest the same for other setters in this class.